engine/verification/utils/fixtures.go

```diff
@@ -229,6 +229,7 @@ func CompleteExecutionResultFixture(t *testing.T, chunkCount int, chain flow.Cha
 	// container block is the block that contains the execution receipt of reference block
 	containerBlock := unittest.BlockWithParentFixture(referenceBlock.Header)
 	containerBlock.Payload.Receipts = []*flow.ExecutionReceipt{receipt}
+	containerBlock.Header.PayloadHash = containerBlock.Payload.Hash()
```

Comment:
In the context of a verification node, a container block is a block that contains some execution receipts. This fixture builds up the container block that contains an execution receipt for sake of testing. 

We need to recompute the payload hash on the header after we include the receipt.